### PR TITLE
mate: add page

### PR DESCRIPTION
**Pull request by superatomic**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** `mate 2.13.3 (Oct 12 2021)`

### CODE REVIEW

File: pages/osx/mate.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Open specific files:
+
+`mate {{path/to/file1 path/to/file2 ...}}`
+
+- Specify the filetype of a file:
+
+`mate --type {{filetype}} {{path/to/file}}`
+
+- Open and wait until done editing a specific file:
```
Comment (einverne) [inline pages/osx/mate.md:18]:
```suggestion
- Open and wait until finishing editing a specific file:
```

**Review by einverne — CHANGES_REQUESTED** [review]
Thank you @superatomic  for this new page. I left a comment for you to take a look.

File: pages/osx/mate.md (hunk 1 above)
Comment (superatomic) [reply pages/osx/mate.md:18]: This exact line was taken directly from [the `webstorm` tldr-page](https://github.com/tldr-pages/tldr/blob/da55f621eb9c77d579b4353d78cc4c9bec39c4f3/pages/common/webstorm.md), but I agree that "finishing" is better than "done".

I think "finished" might be a better form of the word than "finishing" to use here, though.

**Review by einverne — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
LGTM :)
Thanks @superatomic

**Pull request merged**